model/src/main/kotlin/config/ScannerConfiguration.kt

```diff
@@ -51,6 +51,11 @@ data class ScannerConfiguration(
      */
     val createMissingArchives: Boolean = false,
 
+    /**
+     * Mappings from licenses returned by the scanner to valid SPDX licenses.
```

Comment:
You could add here the information from the discussion above, for example: "Note that these mappings are only applied in new scans, stored scan results are not affected."

Reply:
Added the note to the JavaDoc and additionally to the comment in the `reference.conf`.